### ISSUE TITLE
The DRM on Zygor is crazy

### ISSUE THREAD

**Issue by adamk33n3r** [tieonlinux/ZygorDownloader#87]
I just spent a couple hours going through it and it's so funny to me that they spent all this effort in
1. creating a drm/checksum type system
2. obfuscating it behind the facade of "animations"
thereby making it fairly difficult to
1. find where the DRM actually is
2. figure out what checksums you need to have in your `License.lua`
when you can literally just comment out the `ZGV:NeedsAnimatedPopup` call in `Guide.lua` and it bypasses the whole thing lol

**Comment by mazmar** [comment]
i tried your hint but then the guides were broken. any idea what might be wrong?

btw did you mean commenting the whole if condition here?

```
if not path:find("SHARED") and ZGV:NeedsAnimatedPopup(guide) then
	ZGV.AnimatePopup = true
	return nil
end
```

so i ended up with
```
-- if not path:find("SHARED") and ZGV:NeedsAnimatedPopup(guide) then
--   ZGV.AnimatePopup = true
--   return nil
-- end
```

**Comment by adamk33n3r** [comment]
Yes that's what you need to comment out. As for the guides being broken, I think there was something else I had to do to prevent it trying to load gold guides. I can try to look after work but it might be hard to find what I did. I could probably just zip up my files and upload it here if all else fails.
